zenith_utils/src/postgres_backend.rs

```diff
@@ -0,0 +1,147 @@
+// Server-side Postgres connection, as limited as we need.
```

Comment:
Some instructions on how to use this module would be nice. Use `//!` for module comments.

Reply:
done